java/Nikiforov-Oleksandr/src/com/zoolatech/lecture5/tasks/_5/KthLargest.java

```diff
@@ -0,0 +1,25 @@
+package com.zoolatech.lecture5.tasks._5;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class KthLargest {
+    private ArrayList<Integer> arrayList;
+    private final int k;
+    private int arrSize = 0;
```

Comment:
why don't you use the method `arrayList.size()`?

Reply:
All issues fixed